lib/util/state.js

```diff
@@ -163,7 +164,7 @@ export function getErrorMessage(error, intl) {
         id="util.state.noTripFoundWithReason"
         values={{
           noTripFound: noTripFoundText,
-          reason: <FormattedMessage id="util.state.noTripFoundReason" />
+          reason: <FormattedMessage id="util.state.noTripFoundReason" values={{strong:Strong}}/>,
```

Comment:
This wasn't prettified?

Reply:
Not sure what happened there! Repaired in a26a1d74f96c8a675361ddf0b58bb0e4b3c9323a